get-started/what-is-shiny.qmd

```diff
@@ -0,0 +1,160 @@
+
+## Batteries included {#batteries-included}
+
+Shiny for Python comes fully equipped with everything you need to build a dashboard right out of the box, including a rich set of [input](/components/#inputs) and [output](/components/#outputs) [components](/components/) so you can quickly build interactive applications.
+
+[Layout](/layouts/) options let you organize your UI efficiently, while built-in [theming](/docs/ui-customize.qmd) (including [dark mode](/components/inputs/dark-mode)) ensures your app looks great with minimal effort.
+
+Already have a brand guideline?
+You can use [brand.yml](https://posit-dev.github.io/brand-yml/) to apply consistent branding, colors, and logos across your application.
+
+<div class="btn-links">
+  <a href="/components#inputs" class="btn-link-large">
+    Inputs
+    <i class="bi bi-sliders primary" role="img" aria-hidden="true"></i>
+  </a>
+
+  <a href="/components#outputs" class="btn-link-large">
+    Outputs
+    <i class="bi bi-bar-chart-line-fill primary" role="img" aria-hidden="true"></i>
+  </a>
+
+  <a  href="/layouts" class="btn-link-large">
+    Layouts
+    <i class="bi bi-layout-text-window-reverse primary" role="img" aria-hidden="true"></i>
+  </a>
+</div>
+
+## Reactivity {#reactivity}
+
+At the heart of Shiny is [reactivity](/docs/reactive-foundations.qmd), a system that automatically updates [outputs](/components/#outputs) when [inputs](/components/#inputs) change for seamless interactivity, without callbacks.
+
+Shiny's reactive engine avoids unnecessary computations by only re-calculating the outputs whose inputs have changed, making Shiny ideal for fast data-driven applications, enabling live updates for [charts, tables](/components/#outputs), and [reports](https://quarto.org/docs/interactive/shiny/) with minimal effort.
+
+Below is a Shiny application where you have the output text react to _both_ input slider values
+and the second row of text _only_ reacts to the first slider value.
+
+```{shinylive-python}
+#| standalone: true
+#| components: [editor, viewer]
+#| layout: vertical
+#| viewerHeight: 250
+
+from shiny import reactive
+from shiny.express import ui, render, input
+
+# input slider 1
+ui.input_slider("slider1", "Slider 1", min=1, max=50, value=20)
+
+# input slider 2
+ui.input_slider("slider2", "Slider 2", min=1, max=50, value=40)
+
+# output reacts to both sliders
+@render.text
+def both_sliders_output():
+    return f"You have set Slider 1: {input.slider1()}, Slider 2: {input.slider2()}"
+
+# output that only reacts to slider 1
+@render.text
+def result():
+    return f"The value for slider 1 squared: {input.slider1()**2}."
+```
+
+## Templates {#templates}
+
+Hit the ground running with `shiny create` to load a [starter template](/templates/) for common use cases, like [data dashboards](templates/dashboard/), [applications](/templates/database-explorer/), [streaming](/templates/monitor-database/) updates, or [data entry](/templates/survey/).
```

Comment:
The templates section doesn't feel like it fits with the rest of "What is Shiny" because everything else gives a quick intro to a concept. The value of this section is to show a full dashboard; secondarily to link to templates. Maybe you can re-align this section so it fits?